guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
 </div>
 
-## Inputting an address
+Since users cannot control how an address or invoice is presented to them, wallets should be highly flexible in terms of input options and interoperability.
 
-<div class="center" markdown="1">
-{% include image.html
-   image = "/assets/images/guide/payments/send/input-address.png"
-   retina = "/assets/images/guide/payments/send/input-address@2x.png"
-   alt-text = "Address input field prompting the sender to paste the address"
-   width = 400
-   height = 219
-   layout = "float-right-desktop"
+Payment information can be shared in many formats and over diverse communication channels. Each has its own advantages and limitations. More details on the [payment request formats]({{ '/guide/payments/send/payment-request-formats/' | relative_url }}) page.
+
+## Presenting payment requests
+
+Once the application has registered the payment request, it should provide clear instructions and options to the user.
+
+<div class="image-slide-gallery">
+
+{% for item in page.imagesEntryScreens %}
+
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
 %}
+{% endfor %}
+
+</div>
+
+## Manual payment initiation
+
+When responding an invoice that contains all relevent information, the user can quickly review and approve it. In other scenarios, it may be required to manually enter or edit various details.
 
-Once you have gotten the address, it's time to enter the payment details. Bitcoin transactions are irreversible, so both the sender and receiver should take great care in correctly sharing and inputting addresses.
+<div class="image-slide-gallery">
 
-**QR Code** -- Camera access will need to be granted to your Bitcoin application to enable scanning of QR Codes. Once the camera detects a valid address in the QR Code, it should automatically fill the address field.
+{% for item in page.imagesInfo %}
 
-**Copy/Paste** -- When the sender receives the address or payment link as text, your application can detect a valid address in the clipboard and prompt the sender to press a button to paste it.
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
+%}
+{% endfor %}
 
 </div>
```

Comment:
I'd recommend adding a 'note' and 'tags' sub-sections to this section and explain them in more detail. Fee's could also be briefly mentioned as it relates to manual configuration then link to the next section on fees for the deeper dive.

Reply:
Added a "meta-data" section with a bit of background on why this is useful.

Reply:
Yep great addition!